src/tools/log-parser-rs/src/args.rs

```diff
@@ -0,0 +1,86 @@
+use clap::{Parser, ValueEnum};
+use std::path::PathBuf;
+
+#[derive(Parser, Debug)]
+#[command(name="kata-log-parser", author, version, about, long_about = None)] // Read from `Cargo.toml`
+pub struct Cli {
+    pub input_file: PathBuf,
+
+    #[arg(short, long)]
+    pub output_file: Option<PathBuf>,
+
+    #[arg(short, long, help = "check log files and only display output on error")]
+    pub check_only: bool,
+
+    //TODO NOTE:(Not sure if this is still required, or why it needs output file set. (is only here
+    //because it was an arg for the go version.)
+    #[arg(
+        short,
+        long,
+        help = "display debug information (requires '--output-file')",
+        requires = "output_file"
+    )]
+    pub debug: bool,
```

Comment:
Im not sure what more debug info I could display, I already pass out all errors to sdterr unless quiet is set, and I dont know why debug reqires output file. (in the go version, that is.) Unless you really want to keep all the CLI options the go version had, I think Ill just remove this?